cogs/commands.py

```diff
@@ -366,7 +366,13 @@ async def pipsearch(self, ctx, term, order: lambda string: string.lower() = 'rel
                 em = discord.Embed(title=f"Searched for {term}", description=f"No [results]({search}) found.")
                 em.colour = discord.Colour.red()
             await ctx.send(embed=em)
-    
+            
+    @commands.command(name="codeblock")
+    async def cb(self, ctx, *, text):
+        """Puts your code into a copyable codeblock"""
+        await ctx.message.delete()
+        await ctx.send(f'>>> \```\n{text}\n```')
```

Comment:
Why `` \` `` instead of `` ` ``? And you should add ``py`` after the first `` ``` `` so it adds syntax highlighting.

Reply:
> Why `` \` `` instead of `` ` ``? And you should add `py` after the first `` ``` `` so it adds syntax highlighting.

Cuz it will then it will be copyable :)

Reply:
or it will be like real blocks, which the meaning of this pr is probably make it copyable.